app/views/shared/_footer.html.erb

```diff
@@ -0,0 +1,44 @@
+<footer>
+  <div class="container">
+    <div class="footer-top bt">
+      <ul class="footer-links">
+        <li><%= link_to t('menu.innovateForpost'), '#' %></li>
```

Comment:
I suggest not to worry about I18n and focus on ruby/rails.

just a simple PR with code would be enough for everyone, but not in the every PR